s3-private/s3.tf

```diff
@@ -40,6 +40,16 @@ resource "aws_s3_bucket_acl" "main-bucket-data-acl" {
   acl    = "private"
 }
 
+# https://aws.amazon.com/about-aws/whats-new/2021/11/amazon-s3-object-ownership-simplify-access-management-data-s3/
+# https://github.com/hashicorp/terraform-provider-aws/issues/22271
+resource "aws_s3_bucket_ownership_controls" "main" {
```

Comment:
@swiknaba  
I keep getting an error when creating a s3 private bucket (which hasn't happened before). 
This solution is suggested by the issues thread but doesn't solve the problem completely (I have to update the permission in the  console to completely get rid of the error)